PHPCompatibility/Sniffs/InitialValue/NewConstantArraysUsingDefineSniff.php

```diff
@@ -88,8 +90,11 @@ public function process(File $phpcsFile, $stackPtr)
             $targetNestingLevel = \count($tokens[$secondParam['start']]['nested_parenthesis']);
         }
 
-        $array = $phpcsFile->findNext([\T_ARRAY, \T_OPEN_SHORT_ARRAY], $secondParam['start'], ($secondParam['end'] + 1));
-        if ($array !== false) {
+        $array = $phpcsFile->findNext(Collections::$arrayTokensBC, $secondParam['start'], ($secondParam['end'] + 1));
+        if ($array !== false
+            && ($tokens[$array]['code'] === \T_ARRAY
+            || Arrays::isShortArray($phpcsFile, $array) === true)
```

Comment:
Indenting the part with the or statement might help readability

Reply:
Agreed. Fixed up. Will have to wait for Travis to get their act together before this can be merged now though as the build won't run.